src/Compilers/CSharp/Test/Emit2/Semantics/ParamsCollectionTests.cs

```diff
@@ -660,6 +660,333 @@ void verify(MetadataReference comp1Ref)
             }
         }
 
+        [Fact]
+        public void LanguageVersion_03_DelegateNaturalType()
+        {
+            var src1 = @"
+public class Params
+{
+    static public void Test1(params System.ReadOnlySpan<long> a) {}
```

Comment:
Do we have any test for a scenario like the following? Would such a test be useful? No need to take any action in this PR, just want to have the test plan item written down if applicable.

```cs
// comp1
public delegate void Del(params IEnumerable<int> e);
public static void Test1(Del del) { }

// comp2
public static void Main()
{
    Test1(e => { });
}
```

Reply:
> Would such a test be useful?

During an offline conversation we arrived to the answer that it would be interesting to verify whether any language version diagnostics going to be reported on the `Test1` invocation in the second conversation. Since the scenario is not primarily related to the changes made in this PR, I am not going to add a test like this here. But, I will incorporate it to one of my next PRs.